crates/cli_bin/tests/apply.rs

```diff
@@ -2015,3 +2015,101 @@ fn ignores_file_in_grit_dir() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn language_option_apply() -> Result<()> {
+    // Keep _temp_dir around so that the tempdir is not deleted
+    let (_temp_dir, dir) = get_fixture("simple_python", false)?;
+
+    // from the tempdir as cwd, run init
+    run_init(&dir.as_path())?;
+
+    // from the tempdir as cwd, run marzano apply
+    let mut apply_cmd = get_test_cmd()?;
+    apply_cmd.current_dir(dir.as_path());
+    // apply_cmd.current_dir(basic_path);
+    apply_cmd.arg("apply").arg("--force").arg("pattern.grit").arg("--language").arg("python");
```

Comment:
we only want this feature for inline patterns. If the pattern is defined in a file we should log a warning that the option was unused